replica/backup.go

```diff
@@ -186,13 +237,43 @@ func preload(d *diffDisk) error {
 				d.location[j] = 0
 			}
 		}
+		if d.UserCreatedSnap[i] {
+			userCreatedSnapIndx = byte(i)
+		}
 		generator := newGenerator(d, f)
 		for offset := range generator.Generate() {
+			if d.location[offset] != 0 {
+				//We are looking for continuous blocks over here.
```

Comment:
Add comment regarding the need for `userCreatedSnapIndx`

Reply:
This has been replaced with already present snapIndx.